egs2/aishell2/asr1/local/data.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=1
+stop_stage=100
+
+log "$0 $*"
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+. ./path.sh
+. ./cmd.sh
+. ./db.sh
+
+
+if [ -z "${AISHELL2}" ]; then
+  log "Error: \$AISHELL2 is not set in db.sh."
+  exit 2
+fi
+
```

Comment:
Could you mention somewhere to show where the AISHELL2 data can be requested? That could be very helpful for those interested in doing the recipe.